src/ytdl_sub/entries/variables/kwargs.py

```diff
@@ -51,8 +51,8 @@ def _(key: str, backend: bool = False) -> str:
 SPLIT_BY_CHAPTERS_PARENT_ENTRY = _("split_by_chapters_parent_entry", backend=True)
 COMMENTS = _("comments", backend=True)
 UID = _("id")
-EXTRACTOR = _("extractor")
-IE_KEY = _("ie_key")
+EXTRACTOR = _("extractor_key") # EXTRACTOR is saved in the info.json as extractor_key
```

Comment:
`extractor` is its own field too - let's instead add a new one called `EXTRACTOR_KEY` and use it base_entry.py